packages/dapp/components/monitor/price-monitor.tsx

```diff
@@ -35,7 +35,7 @@ const fetchPrices = async (
   ]);
 
   const metaPoolGet = async (i1: BigNumber, i2: BigNumber): Promise<BigNumber> => {
-    return await metaPool["get_dy_underlying(int128,int128,uint256)"](i1, i2, ethers.utils.parseEther("1"));
```

Comment:
This seems like a significant change. Was this broken?

Reply:
Yes, somehow the `dapp` build suddenly broke down with typescript error similar to `Can not find get_dy_underlying(int128,int128,uint256)`

Anyway the following lines are equal:
```
metaPool["get_dy_underlying(int128,int128,uint256)"](i1, i2, ethers.utils.parseEther("1"))
metaPool.get_dy_underlying(i1, i2, ethers.utils.parseEther("1"))
```